services/draft/driving/web-mvc/src/main/java/nettee/draft/driving/web/dto/DraftCommandDto.java

```diff
@@ -50,7 +51,10 @@ public record DraftUpdateCommand(
     @Builder
     public record DraftCommandResponse(
             Draft draft
-    ) {
+    ) {}
 
-    }
+    @Builder
+    public record DraftImageCreateResponse(
+            DraftImage draftImage
```

Comment:
응답 메시지이므로 이름을 단순화하면 좋겠습니다.

```suggestion
            DraftImage image
```

이러면 이렇게 응답할 수 있어 보입니다:

```json
{
  "image": {
    "url": "...",
    ...
  }
}
```